app/repository/database.py

```diff
@@ -0,0 +1,38 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, Session
+from sqlalchemy import create_engine
+from datetime import datetime
+import uuid
+import pytz
+
+DEFAULT_TIMEZONE_REGION = "America/Sao_Paulo"
+
+engine = create_engine("sqlite:///app.db")
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+
+Base = declarative_base()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+    try:
+        yield db
+    except:
+        db.rollback()
+    finally:
+        db.close()
```

Comment:
At this point we `yield` a dependency to be used at the whole project, where for EVERY request a new [Session](https://docs.sqlalchemy.org/en/14/orm/session_basics.html) will be created and closed. This doesn't helps directly on the data race condition problem but intends to help us on concurrency and escalation one